packages/blueprints-integration/src/content.ts

```diff
@@ -18,6 +18,7 @@ export enum SourceLayerType {
 	TRANSITION = 13,
 	// LIGHTS = 14,
 	LOCAL = 15,
+	EVS = 16,
```

Comment:
This is my only real objection: I don't think we should have a SourceLayerType called `EVS`, the same way we don't have a SourceLayerType called `LIVEU` for remote sources. The SourceLayerType should reflect the editorial purpose of this source and not it's techical implementation. In my personal opinion, a replay input (in this case, an EVS machine) falls into the "LOCAL" category, being a local presentation source that is not a camera or a full screen graphic. Then it would fall into the same category as a laptop with a PowerPoint presentation running or a Weather graphics system. I think it would be perfectly fine to just remove the special styling for `.evs`, use the new color for `$segment-layer-background-local` (since we haven't used `LOCAL` anyway) and rename `EvsLayerItemRenderer` to `LocalLayerItemRenderer`

Reply:
This has the side-effect that if we reuse LOCAL for anything else later, those parts will also have the purple background color that is meant to be used for EVS parts now.
The need to denote the specific EVS channel (using the color mark) is also not something that would be relevant for a laptop with a Powerpoint presentation. Since the UI color is optional, that is probably less relevant.

I don't have a particularly strong opinion on it, but it's worth considering.